src/AWSS3.jl

```diff
@@ -865,7 +884,9 @@ function s3_list_versions(aws::AbstractAWSConfig, bucket, path_prefix=""; kwargs
     return versions
 end
 
-s3_list_versions(a...; b...) = s3_list_versions(global_aws_config(), a...; b...)
+function s3_list_versions(bucket, path_prefix=""; kwargs...)
+    return s3_list_versions(global_aws_config(), bucket, path_prefix; kwargs...)
+end
```

Comment:
Avoid redefining default values:
```suggestion
function s3_list_versions(args...; kwargs...)
    return s3_list_versions(global_aws_config(), args...; kwargs...)
end
```